src/app/app.routes.ts

```diff
@@ -1,10 +1,12 @@
 import { NgModule } from '@angular/core';
-import { CommonModule } from '@angular/common';
+import { RouterModule, Routes } from '@angular/router';
+import { LoginComponent } from './login/login.component';
 
+const routes: Routes = [
+  { path: 'login', component: LoginComponent }
+];
 @NgModule({
```

Comment:
Once again: The file can be simplified, module is not necessary here.
So the code can be just:

```
const router: Routes = [
...
];

export const routes: ModuleWithProviders = RouterModule.forRoot(router);

```